src/components/results.js

```diff
@@ -0,0 +1,54 @@
+import React, {Component} from 'react';
+import { View, Text, StyleSheet } from 'react-native';
+
+class ResultSuccess extends Component {
+    render() {
+        return (
+        <View style={styles.container}>
+            <Text style={styles.text}>Hello Results Screen!</Text>
+        </View>
+    )}
+}
+
+class WikiDidMiss extends Component {
+    render() {
+        return (
+        <View style={styles.container}>
+            <Text style={styles.text}>Hello wikiMiss Screen!</Text>
+        </View>
+    )}
+}
+
+
+class BarcodeDidMiss extends Component {
+    render() {
+        return (
+        <View style={styles.container}>
+            <Text style={styles.text}>Hello barcodeMiss Screen!</Text>
+        </View>
+    )}
+}
+
+export default class ResultsView extends Component {
+    render() {
+        return (
+            <View style={{flex:1}}>
+                {this.props.didWikiMiss ? <WikiDidMiss /> :
+                 this.props.didBarcodeMiss ? <BarcodeDidMiss />:
+                <ResultSuccess />}
+            </View>
+        )}
+}
+
+const styles = StyleSheet.create({
+    container: {
+        flex: 1,
+        alignItems: 'center',
+        justifyContent: 'center',
+    },
+

```

Comment:
We will want to work on font and general design sometime, but that will be a much later priority. Since we only have one person on this team, the focus should be more on the functionality. Not that you are focusing on it, I'm just clarifying for the both of us.